src/widgetastic_patternfly/__init__.py

```diff
@@ -2555,7 +2556,7 @@ def read(self):
         Returns:
             :py:class:`dict` complete data on chart
         """
-        return self._get_data(self._elements.values())
+        return self._get_data(list(self._elements.values()))
```

Comment:
a iterator is fine here, why list it?

Reply:
Was just being explicit where the called method might be expecting a list. You're right though, `_get_data` is iterating over it and this is redundant.